### PR TITLE
add LOLIN D32 S2 Board

### PR DESCRIPTION
**Pull request by wemos**
add LOLIN D32 S2 Board

### CODE REVIEW

**Comment by ladyada** [comment]
we cannot accept adding a port that re-use a USB VID/PID

please purchase a USB VID and assign one to your product

File: ports/esp32s2/boards/lolin_d32_s2/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,18 @@
+USB_VID = 0x239A
+USB_PID = 0x80AA
```
Comment (ladyada) [inline ports/esp32s2/boards/lolin_d32_s2/mpconfigboard.mk:2]: change VID and PID to your own

**Review by ladyada — CHANGES_REQUESTED** [review]

**Comment by tannewt** [comment]
Here is info on getting a Vendor ID: https://www.usb.org/getting-vendor-id

**Comment by tannewt** [comment]
@wemos Are you planning on updating this with a valid USB PID?

**Pull request closed without merging**